### PR TITLE
change indent to sensible value

### PR DESCRIPTION
**Pull request by BurntSushi**

### CODE REVIEW

**Review by MichaReiser — APPROVED** [review]
Not sure if I agree with the commit message ;)

**Review by zanieb — APPROVED** [review]
🚲

File: .editorconfig [hunk 1]
```diff
@@ -6,4 +6,4 @@ trim_trailing_whitespace = true
 end_of_line = lf
 indent_style = space
 insert_final_newline = true
-indent_size = 2
\ No newline at end of file
+indent_size = 4
```
Comment (AlexWaygood) [inline .editorconfig:9]: Or do [what we do at ruff](https://github.com/astral-sh/ruff/blob/20217e9bbda5434288d96454e90cb0846c1f330f/.editorconfig#L11-L14)?

```suggestion
indent_size = 2

[*.{rs,py,pyi}]
indent_size = 4
```

**Review by AlexWaygood — APPROVED** [review]

File: .editorconfig (hunk 1 above)
Comment (BurntSushi) [reply .editorconfig:14]: Fair enough. PR updated to just be a copy of what's in Ruff.

**Pull request merged**